sdcm/cluster_k8s/__init__.py

```diff
@@ -2648,7 +2632,9 @@ def connection_bundle_file(self) -> Path | None:
         if bundle_file := super().connection_bundle_file:
             return bundle_file
 
-        bundle_cmd_output = self.k8s_cluster.kubectl(
+        # TODO: support multiDC case
```

Comment:
This depends on operator implementation

It was discussed there would be one bundle that has all the regions in it...

But I don't think it's yet implementef